fawltydeps/packages.py

```diff
@@ -499,7 +528,11 @@ def setup_resolvers(
         mapping_paths=custom_mapping_files or set(), custom_mapping=custom_mapping
     )
 
-    yield LocalPackageResolver(pyenv_srcs)
+    if pyenv_srcs:
+        yield LocalPackageResolver(pyenv_srcs)
```

Comment:
Nit: If we give empty `pyenv_src` then `LocalPackageResolver` will not produce any mapping. For the consistency sake I think it would be better to drop this condition here. We have conditional resolver usage in case of 4a and 4b, and 3 but they arer governed by explicitly provided flags.
```suggestion
    yield LocalPackageResolver(pyenv_srcs)
```

Reply:
Sure, I wanted to skip the creation of an unnecessary object, but as we have no indication of a performance problem here, I agree that clarity/consistency is more important. 👍🏻